Jellyfin.Server/Startup.cs

```diff
@@ -46,7 +52,20 @@ public void ConfigureServices(IServiceCollection services)
             services.AddCustomAuthentication();
 
             services.AddJellyfinApiAuthorization();
-            services.AddHttpClient();
+
+            services
+                .AddHttpClient(NamedClient.Default, c =>
+                {
+                    c.DefaultRequestHeaders.UserAgent.Add(new ProductInfoHeaderValue(_applicationHost.Name.Replace(' ', '-'), _applicationHost.ApplicationVersionString));
+                })
+                .ConfigurePrimaryHttpMessageHandler(x => new DefaultHttpClientHandler());
+
+            services.AddHttpClient(NamedClient.MusicBrainz, c =>
+                {
+                    c.DefaultRequestHeaders.UserAgent.Add(new ProductInfoHeaderValue(_applicationHost.Name.Replace(' ', '-'), _applicationHost.ApplicationVersionString));
+                    c.DefaultRequestHeaders.UserAgent.Add(new ProductInfoHeaderValue($"({_applicationHost.ApplicationUserAgentAddress})"));
```

Comment:
https://docs.microsoft.com/en-us/dotnet/api/system.net.http.headers.productinfoheadervalue.-ctor?view=netcore-3.1#System_Net_Http_Headers_ProductInfoHeaderValue__ctor_System_String_

It's the ctor for adding a comment. Is that what you wanted?

Reply:
You can use ParseAdd instead of Add.

Reply:
`The format of value 'team@jellyfin.org' is invalid.`
Looks like it would still need to be wrapped in parenthesis